shard_to_single_vcf.py

```diff
@@ -0,0 +1,65 @@
+## THIS SCRIPT NEEDS SSDs!!!!
+import argparse
+
+import hail as hl
+from gnomad.utils.slack import try_slack
+
+
+def main(args):
+
+    hl.init(log="/vcf_write.log", default_reference=f"{args.build}")
+
+    mt = hl.import_vcf(
+        f"{args.genotypes_vcf}", force_bgz=True, find_replace=("nul", ".")
+    )
+    filters_ht = hl.import_vcf(
+        f"{args.filtered_vcf}", force_bgz=True, find_replace=("nul", ".")
+    ).rows()
+    mt = mt.annotate_rows(filters=filters_ht[mt.row_key].filters)
+    hl.export_vcf(
+        mt,
+        f"gs://seqr-datasets/v02/{args.build}/RDG_{args.data_type}_Broad_{args.data_source}/v{args.version}/RDG_{args.data_type}_Broad_{args.data_source}.vcf.bgz",
+    )
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
```

Comment:
this is definitely a small thing. not sure if you need a newline here, but probably good to be consistent between this and main